src/openrct2/world/map_generator/PngTerrainGenerator.cpp

```diff
@@ -236,7 +236,7 @@ namespace OpenRCT2::World::MapGenerator
         if (settings->smoothTileEdges)
         {
             // Set the tile slopes so that there are no cliffs
-            smoothMap(settings->mapSize, smoothTileWeak);
+            smoothMap({ mapWidth, mapHeight }, smoothTileWeak);
```

Comment:
`mapWidth` and `mapHeight` should be flipped here like in the `MapInit` call above:

<img width="2320" height="1240" alt="Image" src="https://github.com/user-attachments/assets/1613d2e5-f03a-4a86-bb19-d7137ad5b4bb" />
<img width="2320" height="1240" alt="Image" src="https://github.com/user-attachments/assets/00d3ae4d-9792-402f-a48a-61f99a495c42" />


```suggestion
            smoothMap({ mapHeight, mapWidth }, smoothTileWeak);
```